src/js/components/MultiSelect.js

```diff
@@ -1,14 +1,44 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { DownArrowIcon } from '@/components/Icons/CustomIcons';
-
 export class MultiSelect extends React.PureComponent {
   static propTypes = {
     collections: PropTypes.array.isRequired,
     dropdownLabel: PropTypes.string.isRequired,
   };
+  constructor() {
```

Comment:
Converting this to a functional component is better with Hooks. Wherever possible we should try to use Functional components with Hooks for new implementations